src/main/java/com/asyncy/apis/hub/entities/Repo.java

```diff
@@ -0,0 +1,80 @@
+package com.asyncy.apis.hub.entities;
+
+
+import com.mashape.unirest.http.Unirest;
+import com.mashape.unirest.http.exceptions.UnirestException;
+import org.json.JSONObject;
+import org.springframework.web.util.UriTemplate;
+
+import java.net.URI;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Repo {
+    private final int id;
+    private final String name;
+    private final String full_name;
+    private final String owner;
+    private final String description;
+    private final UriTemplate contentsUrl;
+    private final Boolean _private;
+    private Boolean valid = false;
+
+    public Repo(int id, String name, String full_name, String description, String contents_url, String owner, Boolean _private) {
```

Comment:
Always use unboxed primitives.
```suggestion
     public Repo(int id, String name, String full_name, String description, String contents_url, String owner, boolean _private) {
```

Reply:
Also - no snake case - please use camelCase for all variables.